src/Provider/MapQuest/MapQuest.php

```diff
@@ -128,7 +113,7 @@ public function geocodeQuery(CommonGeocodeQuery $query): Collection
 
         $useGetQuery = true;
 
-        $address = $this->extractAddressFromQuery($query);
+        $address = $query->getData(self::DATA_KEY_ADDRESS);
```

Comment:
Even if it is a one liner, I wanted to keep this as a seperate method, since it it where changes in the code is likely to happen.